### DISCUSSION TITLE
problems with --attr "X-Amz-Meta-Snowball-Auto-Extract=true"

### DISCUSSION THREAD

**Discussion by elfersoto** [minio/minio#17190] [Q&A] [answered]
"Hello to the entire community,

I'm experiencing the following issue:
When I run the command

'mc cp --disable-multipart --attr "X-Amz-Meta-Snowball-Auto-Extract=true" --debug test.tar.gz alias/mybucket/2023/01/'

It doesn't decompress in that path, but it decompresses in 'mybucket/' instead.

Do you know another way to achieve this?"

**Comment by harshavardhana** [comment, marked as answer]
> It doesn't decompress in that path, but it decompresses in 'mybucket/' instead.

Set

```
X-Amz-Meta-Minio-Snowball-Prefix="2023/01/"
```

**Comment by elfersoto** [comment]
I'm trying like this,  mc cp --disable-multipart --attr "X-Amz-Meta-Snowball-Auto-Extract=true" --attr "X-Amz-Meta-Minio-Snowball-prefix=2020/02/" alias/bucket-1/2020/02/filebeat-2020.02.02.tar.gz alias/bucket-2/2020/02/ 
Now i can copy where i want , but doesn't decompress de content, only de complete tar.gz. ¿what I'm doing wrong?

**Reply by harshavardhana** [reply]
You have to specify single --attr argument, the values are comma separated.
